### ISSUE TITLE
After latest update of mods generator is not recognized

### ISSUE THREAD

**Issue by djay-hub** [kwindrem/GuiMods#65]
Hello,
After latest update generator is not recognized and therefore not working ie. manual start...

**Comment by kwindrem** [comment]
It's working here on my test system using a Raspberry PI with Venus OS v2.90~27, GuiMods v6.7 and RpiGpioSetup v3.7.
Please provide a bit more information about your system:
platform (e.g, Cerbo, etc)
firmware version
GuiMods version
Specific issues you are having

**Comment by djay-hub** [comment]
Hello Kevin,
Done fresh clean install of VenusOS v 2.89 with clean install of GUIMods and still not working. Before I installed the GUImMod system was working.
![Screenshot 2022-09-07 at 13-37-38 Remote Console on LAN](https://user-images.githubusercontent.com/61638273/188783491-66e8c0a5-26fa-4554-bc89-6d65fcc4773d.png)
![Screenshot 2022-09-07 at 13-38-05 Remote Console on LAN](https://user-images.githubusercontent.com/61638273/188783555-59164471-d2ea-41c2-b970-db29e5aae8ed.png)

**Comment by peterscarsten** [comment]
Hello,
I've the same issue on my system. In my case it's a Cerbo GX with firmware version 2.89 and GuiMods 6.7

Do you need further information?

Best regards
Carsten

**Comment by djay-hub** [comment]
Hello again,...
Update: 
Reinstalled fresh new VenusOS v 2.87 with GUIMods 6.7 the same thing gen not showing up and is not "active"?

**Comment by kwindrem** [comment]
I found an issue that prevented bus-generator-starter from running in Venus OS v2.80-v2.89. Fixed in GuiMods v6.8

Thanks for reporting.

GeneratorConnector should not be installed along with GuiMods now that GuiMods contains all the GeneratorConnector functionality. BUT having GeneratorConnector and GuiMods installed does not seem to cause the issues you are seeing. 

The GuiMods setup script was supposed to uninstall GeneratorConnector but there was a bug that is fixed in v6.8. So installing GuiMods will insure GeneratorConnector is not installed. It IS possible to manually install GeneratorConnector after installing GuiMods but as I mentioned above I don't think that will cause any issues. It is still a good idea not to install GeneratorConnector after installing GuiMods 6.x
